style.css

```
@@ -81,3 +87,54 @@ body {
 .btn_menu:hover {
 	background-color: #3AC162;
 }
+
+/*Welcome*/
+
+.container {
+	background-image: url('images/overlay.png'), url('images/hero-bg.jpg');
+	background-size: cover;
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	min-height: 500px;
+  padding-top: 180px;
+  background-position: center top;
+}
+
+.welcome_container {
+	max-width: 1150px;
+	padding: 0 10px 0;
+  margin: 0 auto;
+}
+
+.welcome_container h1 {
+	font-size: 48px;
+  font-weight: 700;
+  line-height: 56px;
+  color: #fff;
+  max-width: 500px;
+}
+
+.welcome_container p {
+	color: #eee;
+  margin: 10px 0 0 0;
+  font-size: 24px;
+  font-family: "Raleway", sans-serif;
+  padding-bottom: 40px;
```

Acho que aqui é mais adequado `margin-bottom` ao invés de `padding-bottom`
```suggestion
  margin-bottom: 40px;
```